src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -817,6 +818,13 @@ namespace ScatterKernelDetail{
 
       template < typename T_DeviceAllocator >
       static void* initHeap( T_DeviceAllocator* heap, void* pool, size_t memsize){
+        if( pool == NULL && memsize != 0 )
+        {
+          throw std::invalid_argument(
+            "Scatter policy cannot use NULL for non-empty memory pools.\
```

Comment:
Not sure I understand your point. The constructor of `invalid_argument` only takes a single string literal as input. The sentences were only split into 2 lines to comply with the 80 character limit per line.

I believe the final output should be on a single line to make clear that the sentences together and originate at the same place.